daemon/start.go

```diff
@@ -182,8 +183,11 @@ func (daemon *Daemon) containerStart(ctx context.Context, container *container.C
 	if daemon.UsesSnapshotter() {
 		newContainerOpts = append(newContainerOpts, containerd.WithSnapshotter(container.Driver))
 		newContainerOpts = append(newContainerOpts, containerd.WithSnapshot(container.ID))
+		ctrdimg := containerd.NewImage(daemon.containerdCli, images.Image{
+			Name: container.Config.Image,
+		})
+		newContainerOpts = append(newContainerOpts, containerd.WithImage(ctrdimg))
```

Comment:
This was leftover from the previous code, I must have missed it when I rebased. I think this can be safely removed now since we're fetching the image a different way.

Reply:
Yup, looks like it's fine to remove now. It was previously necessary since we were getting the image from the containerd container object, but we don't need it anymore.